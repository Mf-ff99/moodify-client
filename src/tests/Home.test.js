import React from 'react'
import Home from '../components/Home'

import { shallow } from 'enzyme'

describe(`Home component`, () => {
    it('Renders without crashing', () => {
        shallow(<Home />)
    })
})