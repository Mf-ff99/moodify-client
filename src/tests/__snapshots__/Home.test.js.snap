// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders form approriately 1`] = `
<div
  className="home-view"
>
  <div
    className="quote-generator-container"
  >
    <QuoteGenerator />
  </div>
  <section
    className="home-page"
  >
    <div
      className="mood-display"
      id="mood-display"
    >
      <h1>
        How have you been feeling recently?
      </h1>
      <div
        className="add-mood-filter-btn-container"
      />
      <br />
      <div
        className="chart-container"
      >
        <Chart
          props={Array []}
        />
      </div>
    </div>
    <div
      className="past-moods"
    >
      <Link
        to="/add-mood"
      >
        <button
          className="add-mood-btn"
          type="click"
        >
          Add a mood
        </button>
      </Link>
      <ul
        className="past-moods-list"
        id="mood-list"
      >
        <br />
      </ul>
    </div>
  </section>
</div>
`;
